Dockerfile

```diff
@@ -208,7 +205,8 @@ RUN . $NVM_DIR/nvm.sh \
     && nvm install "$NODEJS_VERSION" \
     && nvm alias default "$NODEJS_VERSION" \
     && nvm use default \
-    && npm install --global npm@$NPM_VERSION bower@$BOWER_VERSION pnpm@$PNPM_VERSION yarn@$YARN_VERSION
+    && npm install --global bower@$BOWER_VERSION corepack@latest \
```

Comment:
commit-msg: Would you mind outlining shortly the motivation to do so?

Furthermore, can you help me understand which version of these tools would be selected for running the tests? ..and whether there is impact on performance?

Reply:
> commit-msg: Would you mind outlining shortly the motivation to do so?
>
> Furthermore, can you help me understand which version of these tools would be selected for running the tests? ..and whether there is impact on performance?

I have updated the commit message and included the information.
Regarding we could pre-install the default versions of the package manager, but in my experience installing them is very fast (just a few seconds).

> It would be IMHO nice if we could "pin" the versions of the tools used for the funTests.

I would suggest to only do this if we see frequent updates of expected results.

Reply:
> I would suggest to only do this if we see frequent updates of expected results.

Hm, it is not only about flakyness. I believe as a user and also as a developer, it is valuable to quickly see against which tool version that thing is actually tested. For development purpose, I would also want to use the exact same tool version as on ORT's CI.  Why do you have reservations against pinning? (I wouldn't even know how this would be done)

Reply:
> Why do you have reservations against pinning? 

For one time constraints to update all the definition files and then also maintenance effort to keep them up to date. Also, I see this as not a big problem while the change itself does solve compatibility issues for users.

Bit I'm also open to keep pre-installing the package managers in defined versions, it just did not seem important to me.

> (I wouldn't even know how this would be done)

It's in the linked docs: https://github.com/nodejs/corepack?tab=readme-ov-file#when-authoring-packages

Reply:
> For one time constraints to update all the definition files and then also maintenance effort to keep them up to date.

I see. I wasn't aware this needs to be done on a per project basis.